src/components/TaskList/TaskList.jsx

```diff
@@ -0,0 +1,72 @@
+//Global Imports
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import { Utils } from '@ukhomeoffice/cop-react-components';
+
+//Local Imports
+import Task from './Task';
+
+import './TaskList.scss';
+
+export const DEFAULT_CLASS = 'hods-task-list';
+
+const TaskList = ({ copRefNum, sections, fieldId, onTaskAction, classBlock, classModifiers, className, ...attrs }) => {
+  const classes = Utils.classBuilder(classBlock, classModifiers, className);
+
+  //TODO state will be retrieved from a document in S3 rather than given in the component definition, covered under COP-9885
+  const numberOfCompleteSections = sections.reduce((totalComplete, currentSection) => {
+    return (
+      totalComplete +
```

Comment:
I think this is easier:
```suggestion
      totalComplete + currentSection.tasks.filter(t => t.state === 'complete').length;
```

Reply:
Nice, updated